ui-components/__tests__/navigation/LanguageNav.test.tsx

```diff
@@ -1,29 +1,54 @@
 import React from "react"
 import { render, cleanup } from "@testing-library/react"
 import { LanguageNav } from "../../src/navigation/LanguageNav"
+import { t } from "../../src/helpers/translator"
```

Comment:
Please push this test upstream as well.

Reply:
I don't think this test is actually applicable to upstream now, since they've gotten rid of the routePrefix logic in the LanguageNav component.